### DISCUSSION TITLE
Ajuda Login

### DISCUSSION THREAD

**Discussion by webdiag** [RamonSilva20/mapos#2382] [General]
Fiz a instalação em dois servidores diferentes mas quando vou acessar fala que as credenciais estão erradas alguém pode ajudar?

**Comment by willph** [comment]
Qual a versão que está utilizando?

Em sex., 12 de abr. de 2024, 16:25, Jefferson ***@***.***>
escreveu:

**Reply by webdiag** [reply]
Estou na ultima

**Comment by willph** [comment]
É atualização ou instalação feita do zero?

Em sáb., 13 de abr. de 2024, 08:13, Jefferson ***@***.***>
escreveu:
